dvc/remote/gdrive/__init__.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import unicode_literals
+
+import os
+
+try:
+    from pydrive.auth import GoogleAuth
```

Comment:
Note that in other remotes we are trying to import such packages dynamically to avoid slowing down startup time. It seems like you could easilly move these imports to where they are actually used, e.g. `drive()` method.